packages/airnode-examples/scripts/choose-integration.ts

```diff
@@ -63,6 +63,15 @@ const questions: PromptObject[] = [
       return '';
     },
   },
+  {
+    // skip for all integrations other than coinmarketcap
+    type: (_prev, values) => (values.integration === 'coinmarketcap' ? 'text' : null),
```

Comment:
I'd rather like the user to copy the secret to `secrets.env` manually and not pollute this script - as this one should remain as "example-agnostic" as possible

Reply:
I thought about that, but a manual copy step is too easily missed, leading to a poor user experience. I think the tradeoff of a busier script but better user experience is favorable. If you really don't want it here, I could create a `prompt` for this integration's `yarn create-airnode-secrets` step so the user can enter it there.

Reply:
I'd rather be more conservative about this - in the future there will be many examples (like this https://github.com/vercel/next.js/tree/canary/examples) and I expect this file to grow more once we support GCP or azure.

> If you really don't want it here, I could create a prompt for this integration's yarn create-airnode-secrets step so the user can enter it there.

Yeah, that seems like a better place.

Reply:
That's fair, but one more thought given the discussion of future integrations: if we expect a number of integrations to use API key authentication then we could retain this step and use a standardized `secrets.env` variable name e.g. `API_KEY`. This would avoid duplicating the request for an API key in each integration's `create-secrets.ts` script. This step would only need to be modified to check if the integration is in a list of ones requiring authentication.

Reply:
It's better if the examples are self contained so I don't think that it's a good idea.

Reply:
I'm still not so sure, but I moved it as you requested. I would appreciate if you could check if what I've done in `check-secrets.ts` is idiomatic.